libtenzir/include/tenzir/atoms.hpp

```diff
@@ -61,6 +61,8 @@ CAF_BEGIN_TYPE_ID_BLOCK(tenzir_atoms, caf::id_block::tenzir_types::end)
   TENZIR_ADD_ATOM(move, "move")
   TENZIR_ADD_ATOM(next, "next")
   TENZIR_ADD_ATOM(normalize, "normalize")
+  TENZIR_ADD_ATOM(package_add, "padd")
+  TENZIR_ADD_ATOM(package_remove, "premove")
```

Comment:
Please use only a `package` atom and combine it with `add` and `remove` atoms as needed.